File: HELLO_PYTHON/day06/mysql_insert.py
import pymysql
conn = pymysql.connect(
    host='127.0.0.1',
    user='root', 
    password='python', 
    db='python', 
    charset='utf8',
    port = 3305
)
cur = conn.cursor()

sql = "INSERT INTO emp VALUES (%s,%s,%s,%s)"
val = ("4","4","4","4")
cur.execute(sql, val)

conn.commit()

cur.close()
conn.close()

File: HELLO_NODE/mysql_insert.js
const mysql = require('mysql');  // mysql 모듈 로드
const conn = {  // mysql 접속 설정
    host: 'localhost',
    port: '3305',
    user: 'root',
    password: 'python',
    database: 'python'
};

let connection = mysql.createConnection(conn); // DB 커넥션 생성
connection.connect();   // DB 접속

console.log("start");
 
sql = "insert into emp (`e_id`,`e_name`,`gen`,`addr`) values (5,5,5,5);";
 /* 쓰레드라서 비동기 방식으로 기다렸다가 마지막으로 출력이 되는 것이다 */
connection.query(sql, function (err, results, fields) { 
    if (err) {
        console.log(err);
    }
    console.log(results.affectedRows);
});
connection.end(); // DB 접속 종료

console.log("end");
